src/cli.ts

```diff
@@ -40,6 +40,11 @@ const args = await yargs(process.argv.slice(2))
       type: "string",
       alias: "l",
     },
+    "include-name-header": {
+      description: "Include a header with the action/workflow name",
+      type: "boolean",
+      alias: "h",
```

Comment:
Maybe better to avoid the `h` since it is a common alias for help. We could use -n from name or switch to 2 letters.

Reply:
Makes sense, thanks I'll update